packages/yew-bootstrap/src/component/pagination.rs

```diff
@@ -0,0 +1,95 @@
+use yew::prelude::*;
+
+
+#[derive(Clone, PartialEq, Eq)]
+pub enum PaginationSize {
+    Large,
+    Normal,
+    Small,
+}
+
+impl Default for PaginationSize {
+    fn default() -> Self {
+        PaginationSize::Normal
+    }
+}
+
+pub struct Pagination {}
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Props {
+    #[prop_or_default]
+    pub callback: Callback<u32>,
+
+    #[prop_or_default]
+    pub current_page: u32,
+
+    #[prop_or_default]
+    pub max_items_per_page: u32,
+
+    #[prop_or_default]
+    pub size: PaginationSize,
+
+    #[prop_or_default]
+    pub total_page_count: u32,
+}
+
+#[function_component(ListPagination)]
```

Comment:
Since this corresponds to Bootstrap's `Pagination`, I would maintain that name. This component is just page buttons. The data for each page would have to be maintained somewhere else. Including `List` in the name suggests to me that this component also maintains the data, which it doesn't